iset.mm

```diff
@@ -99507,6 +99503,47 @@ the previous value and the value of the input sequence (second operand).
      version entirely.  (Contributed by Jim Kingdon, 3-Oct-2022.) $)
   df-seq3 $a |- seq M ( .+ , F ) = seq M ( .+ , F , _V ) $.
 
+  ${
+    $d .+ x y $.  $d F x y $.  $d M x y $.
+    $( Define a general-purpose operation that builds a recursive sequence
+       (i.e. a function on the positive integers ` NN ` or some other upper
+       integer set) whose value at an index is a function of its previous value
+       and the value of an input sequence at that index.  This definition is
+       complicated, but fortunately it is not intended to be used directly.
+       Instead, the only purpose of this definition is to provide us with an
+       object that has the properties expressed by ~ seqf , ~ seq3-1 and
+       ~ seq3p1 .  Typically, those are the main theorems that would be used in
+       practice.
+
+       The first operand in the parentheses is the operation that is applied to
+       the previous value and the value of the input sequence (second operand).
+       The operand to the left of the parenthesis is the integer to start from.
```

Comment:
Another attempt:
> The first argument of `seq` is the integer to start from.  The third argument is the input sequence.  The second argument is the binary operation used to combine the recursively defined values of the resulting sequence with the input sequence.  For instance, `seq M ( + , F )` is the sequence of partial sums of the sequence `F` (starting at the integer `M`) and `seq M ( x. , F )` is the sequence of its partial products.

Reply:
I'm going to leave this for someone else and another pull request. Just numbering the arguments to `seq` is tricky.